### PR TITLE
Bananium/Clown naming consistency for the mineral

### PR DESCRIPTION
**Pull request by ghost**
Renamed 'clown' to 'bananium' for objects related to the mineral, half of them were clown and half were bananium there was no consistency

### CODE REVIEW

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -333,12 +333,6 @@ GLOBAL_LIST_INIT(runed_metal_recipes, list ( \
 		return
 	return ..()
 
-/obj/item/stack/sheet/runed_metal/attack(atom/target, mob/living/user)
```
Comment (FantasticFwoosh) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: Stop please, this doesn't relate to bananium and is a silent revert not even mentioned on the commit, only slightly on the title with no supplied change log.

**Review by ninjanomnom — DISMISSED** [review]
the attack check prevents such things as runed walls and golems, atomize this pr regardless

File: code/game/objects/items/stacks/sheets/sheet_types.dm (hunk 1 above)
Comment (uraniummeltdown) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: it's literally in the commit and the PR desc

**Comment by uraniummeltdown** [comment]
@ninjanomnom done

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**